client/src/pages/CreateContest/CreateContest.tsx

```diff
@@ -23,25 +26,29 @@ export default function CreateContest():JSX.Element {
       title: title,
       description: description,
       prizeAmount: prizeAmount,
-      deadlineDate: deadlineDate,
+      deadlineDate: deadlineDate.format('MMMM Do YYYY h:mm A z'),
       images: images,
     };
 
     addContest(contest);
+    history.push('/discovery');
```

Comment:
Maybe check the response and only when it's successful redirect the user to the dashboard? In case a contest creation fails, redirecting the user to the dashboard is not very user friendly